### ISSUE TITLE
The charge parameters are ignored when docking with aISS

### ISSUE THREAD

**Issue by RaphaelRobidas** [grimme-lab/xtb#825]
**Describe the bug**
Specifying the `--chrg` or `--chrg2` parameters does not set the charge. 

**To Reproduce**
1. Run the docking procedure: `xtb dock Cl.xyz NH3.xyz --chrg -1` 
2. Verify the charge in the calculation setup. It is still `0`:
```
      -----------------------------------------------------------
     |                   =====================                   |
     |                          a I S S                          |
     |                   =====================                   |
     |               S. Ehlert, S. Grimme, C.Plett               |
     |          Mulliken Center for Theoretical Chemistry        |
     |                    University of Bonn                     |
      -----------------------------------------------------------

   * xtb-docking algorithm 6.6.0 (8843059) compiled by 'stahn@majestix' on 2023-02-09

 * started run on 2023/06/23 at 12:58:52.071     
 taking internal default parameters

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : xtb dock Cl.xyz NH3.xyz --chrg -1
          omp threads                :                    16
          coordinate file A          : Cl.xyz
          coordinate file B          : NH3.xyz
          number of atoms A          :                     1
          number of atoms B          :                     4
          charge of molecule A       :                   0.0
          charge of molecule B       :                   0.0
          spin of molecule A         :                     0
          spin of molecule B         :                     0
          first test random number   :      0.81772429240242
```

Files used in this run:
**Cl.xyz**
```
1

Cl 0.0 0.0 0.0
```

**NH3.xyz**
```
4

N          0.41850       -1.05150        0.00000
H          0.75180       -1.99430        0.00000
H          0.75180       -0.58010        0.81650
H          0.75180       -0.58010       -0.81650
```

To my knowledge, this bug occurs with any input file.

**Expected behaviour**
The charge parameter should set the charge.

**Comment by cplett** [comment]
Hi Raphael,
Thank you for pointing that out.
This was a problem of the xtb 6.6.0 release and was fixed with #771. Version 6.6.0 can however read in `.CRHG` and `.UHF` files correctly.

**Comment by MtoLStoN** [comment]
This should be fixed in xTB 6.6.1. I will therefore close this Issue.
